### PR TITLE
Update 2279 integration tests

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** Summary of changes

Addresses [CUMULUS-2279](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2279)

## Changes

* Update cmr-client and integration-test helpers to allow granule concept CMR search

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Adhoc testing
- [ ] Integration tests

### CODE REVIEW

File: packages/cmr-client/src/getConcept.ts [hunk 1]
```diff
@@ -3,35 +3,53 @@ import Logger from '@cumulus/logger';
...
-
   try {
     response = await got.get(conceptLink, { headers });
   } catch (error) {
     log.error(`Error getting concept metadata from ${conceptLink}`, error);
-    return null;
+    return undefined;
   }
 
   if (response.statusCode !== 200) {
```
Comment (Jkovarik) [inline packages/cmr-client/src/getConcept.ts:19]: Handle 404 null return for 'granuleConceptSearch' type

Comment (nemreid) [reply packages/cmr-client/src/getConcept.ts:19]: Does this also need to be done for the `conceptExists` function below?

Comment (Jkovarik) [reply packages/cmr-client/src/getConcept.ts:19]: It is - it handles a 404 by setting response to error.response, then false when the status code (404) is !==200 on L150

Comment (nemreid) [reply packages/cmr-client/src/getConcept.ts:19]: Oh, I see. Right.

**Comment by nemreid** [comment]
As far as I can tell, you have addressed the places throughout the codebase where we make use of the CMR link. I'm going to merge. Thanks for doing this!

**Pull request merged**